### PR TITLE
Updated psimol/configs/atom_properties.yml

### PR DESCRIPTION
**Pull request by macbiel**
Corrected and extended covalent radii
Confirmed metallic radii
Corrected valences
Confirmed atomic masses

### CODE REVIEW

File: psimol/configs/atom_properties.yml [hunk 1]
```diff
@@ -80,8 +83,9 @@ O:
   atomic_number: 8
   valence: 2
   covalent_radius:
-    - 0.66 # single bond
-    - 0.60 # double bond
+    - 0.63
+    - 0.57
+    - 0.53
```
Comment (dav3794) [inline psimol/configs/atom_properties.yml]:
```suggestion

```
Let's skip triple bonded oxygens as unusual cases.

File: psimol/configs/atom_properties.yml [hunk 2]
```diff
@@ -138,18 +150,20 @@ Si:
...
+    - 1.07
+    - 1.02
 
 P:
   name: Phosphorus
   symbol: P
   mass: 31.0
   atomic_number: 15
-  valence: 5
+  valence: 3
```
Comment (dav3794) [inline psimol/configs/atom_properties.yml]:
```suggestion
  valence: 5
```
Phosphorus is organically usually at V oxidation state (in ATP, DNA etc).

File: psimol/configs/atom_properties.yml [hunk 3]
```diff
@@ -158,8 +172,9 @@ S:
   atomic_number: 16
   valence: 2
   covalent_radius:
-    - 1.05 # single bond
-    - 1.02 # double bond
+    - 1.03
+    - 0.94
+    - 0.95
```
Comment (dav3794) [inline psimol/configs/atom_properties.yml]:
```suggestion
```
Similarly as with oxygen, skip unusual triple bonded sulfur, as it's uncommon in organic molecules.

**Comment by dav3794** [comment]
Basically we can remove all cases of triple bondings which have higher or equal covalent radius than double bond - these are usually obtained in specific conditions (as it is physically abnormal to have longer triple than double bond) and we cannot really handle such things correctly while parsing xyz coordinates. 

+ Most importantly, triple bonds for elements from groups 1-2 and 16-17 should be removed. We could also remove double bonds for group 1.

**Comment by dav3794** [comment]
Additionally, we could possibly add remaining atoms up to bizmuth (so we would have gold, platinum, mercury and lead supported as well), but we can skip the f-block.

**Comment by macbiel** [comment]
Removed overlong triple bonds from all elements
Removed triple bonds from elements of groups 1, 2, 16 & 17
Removed double bonds from elements of group 1
Extended the list to include elements of period 6 sans lanthanides, i.e.: Cs, Ba, and Lu to Rn

**Review by dav3794 — APPROVED** [review]
Splendid

**Pull request merged**